src/resources/members/controller.js

```diff
@@ -20,6 +20,102 @@ async function getAllMembers( req, res){
  }
 }
 
+async function addNewMember(req, res){ 
+
+    const memeber = req.body
+    console.log("memeber: ", memeber)
+
+    try { 
+
+        const newMember = await prisma.member.create({
+            data : { 
+                ...memeber
+            }
+        })
+
+        res.json({ theNewMemeber: newMember})
```

Comment:
This means your frontend will be receiving a JSON object that looks like this:

```
{
    theNewMemeber: {
       // properties here
    }
}
```

That looks a little weird. I think you should just pass the new member object into json directly; `res.json(newMember)`

Reply:
oh, you're so right, did that on default mode based on reflex when checking `res` in the postman
I shall update accordingly

Reply:
done, this is soo cool 🤩 I can replay from VS code
once COVID showed up could not leave my house
now, once these features are here, I wont step outside my code editor 🥳